interface/Gruntfile.js

```diff
@@ -96,6 +98,37 @@ module.exports = function ( grunt ) {
     ],
 
     /**
+     * Browserify the ttest library since it's written just for node
+     * Note: this must be run before the copy step (see `build` below).
+     */
+    browserify: {
+      dist: {
+        src: 'node_modules/ttest/hypothesis.js',
+        dest: 'node_modules/ttest/ttest.js',
```

Comment:
That is a really good thought. Thank you for bringing it up, because I didn't think of it!

How about `greenelab.stats.ttest.js`? That sort of mimics the Angular module hierarchy and helps make the connection between this source file and where it ends up.